BatchExamples/DFIRBatch.reb

```diff
@@ -2693,6 +2693,181 @@ Keys:
+        KeyPath: Usoris\Remote Utilities\RManService\Host\Parameters
+        Recursive: true
+        Comment: "Displays artifacts relating to Portable RemoteUtilities Configuration"
+    -
+        Description: RemoteUtilities
+        HiveType: NTUSER
+        Category: Third Party Applications
+        KeyPath: Software\Usoris\Remote Utilities\RManService\Host\Parameters
+        ValueName: General
+        Recursive: false
+        Comment: "Displays artifacts relating to RemoteUtilities Configuration - Base64 decode output"
+    -
+        Description: RemoteUtilities
+        HiveType: NTUSER
+        Category: Third Party Applications
+        KeyPath: Software\Usoris\Remote Utilities\RManService\Host\Parameters
+        ValueName: InternetId
+        Recursive: false
+        Comment: "Displays artifacts relating to RemoteUtilities Configuration - Base64 decode output"
+    -
+        Description: RemoteUtilities
+        HiveType: NTUSER
+        Category: Third Party Applications
+        KeyPath: Software\Usoris\Remote Utilities\RManService\Host\Parameters
+        ValueName: Security
+        Recursive: false
+        Comment: "Displays artifacts relating to RemoteUtilities Configuration - Base64 decode output"
+    -
+        Description: RemoteUtilities
+        HiveType: NTUSER
+        Category: Third Party Applications
+        KeyPath: Software\Usoris\Remote Utilities\RManService\Host\Parameters
+        ValueName: FUSClientPath
+        Recursive: false
+        Comment: "Displays artifacts relating to RemoteUtilities Configuration"       
+# https://jsac.jpcert.or.jp/archive/2023/pdf/JSAC2023_1_1_yamashige-nakatani-tanaka_en.pdf
+
+# Third Party Applications -> Splashtop - https://www.splashtop.com/
+
+    -
+        Description: Splashtop
+        HiveType: SYSTEM
+        Category: Third Party Applications
+        KeyPath: CurrentControlSet\Services\SplashtopRemoteService
+        Recursive: true
+        Comment: "Displays artifacts relating to Splashtop"
+    -
+        Description: Splashtop
+        HiveType: SYSTEM
+        Category: Third Party Applications
+        KeyPath: CurrentControlSet\Services\SSUService
+        Recursive: true
+        Comment: "Displays artifacts relating to Splashtop"
+# https://jsac.jpcert.or.jp/archive/2023/pdf/JSAC2023_1_1_yamashige-nakatani-tanaka_en.pdf
+
+# Third Party Applications -> TeamViewer - https://www.teamviewer.com/en-us/
+
+    -
+        Description: TeamViewer
+        HiveType: SYSTEM
+        Category: Third Party Applications
+        KeyPath: CurrentControlSet\Services\TeamViewer
+        Recursive: true
+        Comment: "Displays artifacts relating to Splashtop"
+# https://jsac.jpcert.or.jp/archive/2023/pdf/JSAC2023_1_1_yamashige-nakatani-tanaka_en.pdf
+
+# Third Party Applications -> TightVNC - https://www.tightvnc.com/
+
+    -
+        Description: TightVNC
+        HiveType: SYSTEM
+        Category: Third Party Applications
+        KeyPath: tvnserver
```

Comment:
CurrentControlSet\Services\tvnserver for keypath? Don’t see tvnserver documented in that location based on the trend document